openrtb_ext/imp_unruly.go

```diff
@@ -1,5 +1,5 @@
 package openrtb_ext
 
 type ExtImpUnruly struct {
-	SiteID int `json:"siteid"`
```

Comment:
You want to keep the old `siteid` entry if you want to keep supporting the old configuration. (Highly recommended for atleast a transition period). So something like:
```
type ExtImpUnruly struct {
	SiteIDOld int `json:"siteid"`
        SiteID int `json:"siteId"`
}
```
And probably have some of each type in your tests to verify that both the old and the new casings work with the current code.

Reply:
Yeah agreed, having both the old configuration and the new casing is what I'd recommend.